File: Src/Campus.Domain.Core/Models/Project.cs
using System.Collections.Generic;

namespace Campus.Domain.Core.Models
{
    public class Project
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string Color { get; set; }
        
        public int StatusId { get; set; }

        public Status Status { get; set; }

        public int UserId { get; set; }

        public AppUser User { get; set; }

        public ICollection<UserTask> Tasks { get; set; }
    }
}

File: Src/Campus.Master.API/UI/src/app/shared/models/task-list/project.ts
export class Project {
  id: number;
  name: string;
  color: string;
  status: number;

  constructor(id: number, title: string, color: string, status: number) {
    this.id = id;
    this.name = title;
    this.color = color;
    this.status = status;
  }
}
